lib/airbrake-ruby/time_truncate.rb

```diff
@@ -9,7 +9,8 @@ module TimeTruncate
     # @param [Time] time
     # @return [String]
     def self.utc_truncate_minutes(time)
-      tm = time.getutc
+      # time is an instance of Float instead of Time on Rails 7.0.0
+      tm = time.is_a?(Float) ? Time.at(time).getutc : time.getutc
```

Comment:
We should rather check for `getutc` existence than for the class of `time`. It would be more sturdy.

Reply:
While I am not sure if it is better, fixed it in f4bdea4.

Reply:
Thanks! Curious to know why it’s not 🙂